lisa/runners/lisa_runner.py

```diff
@@ -776,6 +776,13 @@ def _merge_test_requirements(
                             and hasattr(self, "platform")
                             and self.platform.runbook.ignored_capability
                         ):
+                            self._log.debug(
```

Comment:
It may produce many lines. Use a variable to log the lines only once.

Reply:
It doesn't need to log the information, because it can be found in other lines. We need to log line, when a feature is ignored. So it means the ignored feature is effective.